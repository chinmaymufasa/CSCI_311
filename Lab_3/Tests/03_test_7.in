2
18
5
2
53
5
2
29
5
2
22
5
2
63
5
-1
