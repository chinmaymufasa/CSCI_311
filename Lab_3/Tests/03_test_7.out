18 
18 53 
18 29 53 
18 22 29 53 
18 22 29 53 63 
